cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -23,6 +23,281 @@ Cask Data Application Platform Release Notes
+  Improved error message when stream views are not found.
+
+- `CDAP-4316<https://issues.cask.co/browse/CDAP-4316>`__ -
+  Don't connect to CDAP successfully in CLI when the hostname and port are incorrect.
+
+- `CDAP-3807<https://issues.cask.co/browse/CDAP-3807>`__ -
+  Added fix for case sensitivity handling in DBSink
+
+- `CDAP-3761<https://issues.cask.co/browse/CDAP-3761>`__ -
+  DBSink column names are used to filter input records fields before writing them to DBSink.
+
+- `CDAP-4534<https://issues.cask.co/browse/CDAP-4534>`__ -
+  Added a configuration to exempt certain URLs from authentication.
+
+- `CDAP-2481<https://issues.cask.co/browse/CDAP-2481>`__ -
+  Removed deprecated endpoint to get status of currently running node in a workflow.
+
+- `CDAP-2470<https://issues.cask.co/browse/CDAP-2470>`__ -
+  Added a check to cause a DB (source or sink) pipeline to fail during deployment if the table (source or sink)
+  was not found, or if an incorrect connection string was provided.
+
+- `CDAP-3815<https://issues.cask.co/browse/CDAP-3815>`__ -
+  Fixed an issue where the regex filter for S3 Batch Source wasn't getting applied correctly.
+
+- `CDAP-4000<https://issues.cask.co/browse/CDAP-4000>`__ -
+  Fixed an issue where a stream that has a view could not be deleted cleanly.
+
+- `CDAP-1174<https://issues.cask.co/browse/CDAP-1174>`__ -
+  Prefix start script functions with cdap to prevent namespace collisions
+
+- `CDAP-4678<https://issues.cask.co/browse/CDAP-4678>`__ -
+  Fixed issue where Logs and History list on Hydrator pipeline view is not updating on new runs
+
+- `CDAP-4658<https://issues.cask.co/browse/CDAP-4658>`__ -
+  Added system metadata UI for CDAP entities
+
+- `CDAP-4598<https://issues.cask.co/browse/CDAP-4598>`__ -
+  Adding error message when a required field is not filled up when configuring Hydrator Pipeline
+
+- `CDAP-4083<https://issues.cask.co/browse/CDAP-4083>`__ -
+  Fixed an issue with the documentation of the Data Quality Application's "Unique Value" function.
+
+- `CDAP-4145<https://issues.cask.co/browse/CDAP-4145>`__ -
+  Added documentation of the new view and stream-views feature.
+
+- `CDAP-4141<https://issues.cask.co/browse/CDAP-4141>`__ -
+  Fixed node.js version checking for the cdap.sh script in the cdap sdk.
+
+- `CDAP-4393<https://issues.cask.co/browse/CDAP-4393>`__ -
+  Fixed the bug in Tags search which didnt' allow user to search for all the tags.
+
+- `CDAP-3695<https://issues.cask.co/browse/CDAP-3695>`__ -
+  No failures occurred in the recent builds. Marking as resolved fix.
+
+- `CDAP-4301<https://issues.cask.co/browse/CDAP-4301>`__ -
+  Added support for CDH5.5
+
+- `CDAP-3900<https://issues.cask.co/browse/CDAP-3900>`__ -
+  Return empty list in case of logs are requested for the deleted program runs.
+
+- `CDAP-3542<https://issues.cask.co/browse/CDAP-3542>`__ -
+  Fixed issue when workflow contains fork and program executing on one of the branches fails because of some error,
```

Comment:
Fixed an issue where the failure of a program running in a workflow fork node was causing other programs in the same fork node to remain in the RUNNING state, even after the Workflow was completed.